src/app/_api/external/farcaster.ts

```diff
@@ -66,7 +66,7 @@ export const getFarcasterUser = async (
       if (!response.ok) return null;
       if (response.status !== 200) return null;
       if (!data[walletId]) return null;
-      return data[walletId][0];
+      return data[walletId].filter((v) => !!v.display_name || !!v.pfp_url)[0];
```

Comment:
To avoid type errors (this can return undefined, instead of null) do 

```ts
      return data[walletId].filter((v) => !!v.display_name || !!v.pfp_url).at(0) ?? null;
```